pkg/webui/components/key-value-map/entry.js

```diff
@@ -66,7 +78,7 @@ class Entry extends React.Component {
           placeholder={valuePlaceholder}
           type="text"
           onChange={this.handleValueChanged}
-          onBlur={onBlur}
+          onBlur={this.handleBlur}
```

Comment:
Shouldn't this handler also be applied to both inputs? Overall, this reminds me of the blur solution that we have for the Join EUI prefix input: https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.9/pkg/webui/console/containers/join-eui-prefixes-input/join-eui-prefixes-input.js#L106-L133